styles/AdvancedEditor.css

```diff
@@ -713,14 +729,19 @@ figure .selected-suggestion {
   box-shadow: inset 3px 0 0 var(--inserted-pending-color);
 }
 
-.selected-suggestion[data-track-status='pending'][data-track-op='insert'] .block,
-.selected-suggestion[data-track-status='pending'][data-track-op='insert'] .block,
+.tracking-visible .selected-suggestion[data-track-status='pending'][data-track-op='insert'] .block,
+.tracking-visible .selected-suggestion[data-track-status='pending'][data-track-op='insert'] .block,
 .selected-suggestion[data-track-op='wrap_with_node'][data-track-status='pending'] .block {
   box-shadow: inset 6px 0 0 var(--inserted-pending-color), inset 9px 0 0 var(--inserted-pending-bg-color) !important;
   animation: fadeOutBackground 3s forwards;
   --fade-color: var(--inserted-pending-bg-color);
 }
 
+.selected-suggestion[data-track-status='pending'] .block {
```

Comment:
Please move this above the comment `/* Inserted Block */` and let it cover move cases like:
```
.selected-suggestion[data-track-status='pending'] .block,
.block:has(figure.selected-suggestion),
figure.block:has(.equation.selected-suggestion)
 {
  box-shadow: inset 6px 0 0 black, inset 9px 0 0 lightgray;
  --fade-color: none;
}

```

Reply:
updated